libs/utils/src/lib/format/strings.spec.ts

```diff
@@ -28,7 +29,7 @@ describe('shorten', () => {
     { i: '12345678901234567890', l: 10, o: `123456789${ELLIPSIS}` },
     { i: '12345678901234567890', l: 20, o: `1234567890123456789${ELLIPSIS}` },
     { i: '12345678901234567890', l: 30, o: `12345678901234567890` },
-  ])('should shorten given string by specific limit', ({ i, l, o }) => {
+  ])(`should shorten given string by specific limit`, ({ i, l, o }) => {
```

Comment:
chore (non-blocking): multiple tests with same name is probably not dresired, can we add the parameters in the name or similar to make this easier to debug in future?